src/main/resources/assets/icons/mixin/mixin_mark.svg

```diff
@@ -0,0 +1,215 @@
+       transform="translate(5,5)"
+       id="path2" /></g><g
+     style="display:none;fill:none"
+     id="g12"><circle
+       cx="8"
+       cy="8"
+       r="6.5"
+       fill="#25324d"
+       stroke="#548af7"
+       id="circle1-6" /><path
+       d="m 8.13295,11.5 c 1.47928,0 2.75065,-0.8895 3.07455,-2.16091 H 10.2213 C 9.90229,10.0739 9.11914,10.6057 8.13295,10.6057 6.77936,10.6057 5.80284,9.51796 5.80284,8 c 0,-1.51796 0.97652,-2.60566 2.33011,-2.60566 0.98619,0 1.76934,0.53177 2.08835,1.26657 h 0.9862 C 10.8836,5.3895 9.61223,4.5 8.13295,4.5 6.21859,4.5 4.79248,5.99378 4.79248,8 c 0,2.0062 1.42611,3.5 3.34047,3.5 z"
+       fill="#548af7"
+       id="path1-8" /></g><g
+     id="layer2"
+     style="display:inline"
+     transform="matrix(1.2403168,0,0,1.2403168,-3.8450688,-0.76589678)"><path
+       id="path0"
+       d="M 10.532818,7.9222705 C 10.07829,7.9364795 9.7469362,8.2234886 9.669004,8.6703945 9.650472,8.7766547 9.649751,11.067566 9.6683493,11.181036 c 0.075712,0.465432 0.4199537,0.755668 0.9344787,0.787906 0.06438,0.0039 0.119166,0.0095 0.121798,0.01211 0.0071,0.0071 0.006,1.716705 -0.0013,1.742067 -0.0033,0.01185 -0.0098,0.02161 -0.01435,0.02161 -0.01441,0 -0.184743,0.08355 -0.220536,0.108185 -0.0995,0.06845 -0.174154,0.187001 -0.198454,0.315127 -0.02016,0.106064 -0.01781,0.670644 0.0033,0.765333 0.04722,0.213562 0.197492,0.362845 0.397635,0.395043 0.347875,0.05599 0.617445,-0.07661 0.671568,-0.330332 0.01303,-0.06122 0.01446,-0.824563 0.0013,-0.896346 -0.0256,-0.143527 -0.112114,-0.236684 -0.298212,-0.321218 -0.04989,-0.02266 -0.09463,-0.04794 -0.09935,-0.05619 -0.01237,-0.02161 -0.01231,-1.735819 8e-5,-1.748215 0.0065,-0.0065 0.506253,-0.0085 1.893608,-0.0079 2.063245,0.0013 1.975333,0.0033 2.116582,-0.04393 0.31937,-0.106103 0.536015,-0.334438 0.613967,-0.64707 l 0.01864,-0.07463 V 10.357933 9.5131948 l -0.02269,-0.090368 C 15.514941,9.1374235 15.320495,8.9168879 15.052201,8.8169533 14.93052,8.7716386 14.88398,8.7679061 14.379318,8.7631258 l -0.459617,-0.00458 -0.0094,-0.047148 c -0.0053,-0.025932 -0.01487,-0.080741 -0.02148,-0.1217995 -0.05205,-0.3213936 -0.336951,-0.6027504 -0.668619,-0.6603498 -0.0611,-0.010608 -2.380728,-0.016895 -2.687447,-0.00727 M 13.0709,8.3382183 c 0.28228,0.031105 0.412212,0.1770022 0.424942,0.4771005 0.01466,0.345557 -0.09494,0.524092 -0.353728,0.576112 -0.110348,0.0222 -2.55834,0.02541 -2.6846,0.0033 -0.17091,-0.0296 -0.294183,-0.1391254 -0.345024,-0.3065212 -0.01617,-0.053238 -0.01958,-0.3348702 -0.0046,-0.4045512 0.03464,-0.1643901 0.147024,-0.2863074 0.305854,-0.3317269 0.02456,-0.00701 0.325283,-0.010542 1.184088,-0.013817 0.633161,-0.00262 1.207779,-0.00523 1.27693,-0.00589 0.06915,-0.00131 0.157553,0.00196 0.196451,0.00589"
+       stroke="none"
+       fill="#ebac24"
+       fill-rule="evenodd"
```

Comment:
Looks like only this path is needed to render correctly, there's like 200 lines of fat for no clear reason.

Reply:
Weird that Inkscape was keeping all the hidden layers I used for testing